scripts/test_forwarder.bash

```diff
@@ -0,0 +1,184 @@
+        tar -czf result_files.tar.gz result_*
+        exit 1
+    fi
+}
+
+
+# Function to print a separator line
+print_separator() {
+    echo "#####################################################################"
+}
+
+# Start of the script, redirecting output to `result.txt` while also displaying on the terminal
+print_separator
+echo "# Sekoia.io script that check the status of the Sekoia.io forwarder #"
+print_separator
+
+# Get the region if specified
+LOCATION=${1:-FRA1}
+LOCATION=$(echo "$LOCATION" | tr '[:lower:]' '[:upper:]')
+
+case "$LOCATION" in
+    FRA1)
+        HOST="intake.sekoia.io"
+        ;;
+    FRA2)
+        HOST="fra2.app.sekoia.io"
+        ;;
+    MCO1)
+        HOST="mco1.app.sekoia.io"
+        ;;
+    UAE1)
+        HOST="app.uae1.sekoia.io"
+        ;;
+    # Ajoutez d'autres cas ici
+    *)
+        echo "Unknown location: $LOCATION"
+        tar -czf result_files.tar.gz result_*
+        exit 1
+        ;;
+esac
+
+# Create result folder
+mkdir $RESULT_FOLDER
+echo "- Creating result folder: OK"
+
+# Current timestamp
+echo "- Timestamp: $CURRENT_TIME_ISO" | tee -a $RESULT_FOLDER/$EXEC_FILE
+
+# Current path
+echo "- Current path: `pwd`" | tee -a $RESULT_FOLDER/$EXEC_FILE
+
+# OS
+echo "- OS: `uname -a`" | tee -a $RESULT_FOLDER/$EXEC_FILE
+
+# Get docker-compose.yml and intakes.yml
+echo "- Get docker-compose and intakes files"
+cp docker-compose.yml intakes.yaml $RESULT_FOLDER
+
+# Check if docker is installed and get the version
+if ! command -v docker &> /dev/null
+then
+    echo "- Docker could not be found" | tee -a $RESULT_FOLDER/$EXEC_FILE
+    tar -czf result_files.tar.gz result_*
+    exit 1
+fi
+echo "- Docker version: `docker --version`" | tee -a $RESULT_FOLDER/$EXEC_FILE
+
+# Check if the Docker container is running
+# Get the Docker container ID associated with the image name
+docker_id=$(docker ps | grep "$IMAGE_NAME" | awk '{print $1}')
+
+# Check if the docker_id variable is non-empty, which indicates that the container is running
+if [ -n "$docker_id" ]; then
+    echo "- Container is running: OK" | tee -a $RESULT_FOLDER/$EXEC_FILE
+else
+    echo "- Container is running: KO" | tee -a $RESULT_FOLDER/$EXEC_FILE
+    tar -czf result_files.tar.gz result_*
+    exit 1
+fi 
+
+# Get information from docker inspect
+echo "- Getting container information from docker inspect into inspect.txt"
+docker inspect $docker_id > $RESULT_FOLDER/$INSPECT_FILE
+
+install_packages | tee -a $RESULT_FOLDER/$EXEC_FILE
+
+# Checking outbound flow to intake.sekoia.io
+echo "- Checking if the connection to $HOST:$PORT is opened..."
+if nc -zv "$HOST" "$DST_PORT" &> /dev/null; then
+    echo "- Port $DST_PORT on $HOST is opened." | tee -a $RESULT_FOLDER/$EXEC_FILE
+else
+    echo "- Port $DST_PORT on $HOST is closed." | tee -a $RESULT_FOLDER/$EXEC_FILE
+fi
+
+# Get docker logs
+echo "- Getting container logs"
+docker logs $docker_id &> $RESULT_FOLDER/$LOGS_FILE
```

Comment:
@goudyj will it be appropriate to limit the logs output to the last 10 000 lines?